backend-rust/src/graphql_api/token.rs

```diff
@@ -75,6 +78,61 @@ impl Token {
     async fn contract_address_formatted(&self, ctx: &Context<'_>) -> ApiResult<String> {
         Ok(format!("<{},{}>", self.contract_index, self.contract_sub_index))
     }
+
+    async fn accounts(
```

Comment:
Not sure what the current behavior is in the API, but we are including accounts with zero balance here, as long as they have been involved with the token at some point, not sure whether this is the desired behavior.

If we choose to filter out the accounts with zero balance then we cannot do the `index_per_token` trick...

Reply:
I exclude tokens with a 0 balance now while still displaying tokens with a negative balance.
Making "buggy" negative balances visible helps smart contract developers to debug their contracts.